### PR TITLE
Moved download-missing-blocks script to archive from rosetta

### PR DESCRIPTION
**Pull request by dkijania**
Moved download-missing-blocks.sh to archive debian and reference it in daemon so rosetta docker will still have an access to it.

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

File: scripts/archive/download-missing-blocks.sh [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        shift 2;
+      ;;
+      -c | --blocks-bucket )
+        BLOCKS_BUCKET=${2:?$error_message}
+        shift 2;
+      ;;
+      * )
+        echo -e "${RED} !! Unknown option: $1${CLEAR}\n";
+	  		echo "";
+				initial_help;
```
Comment (emberian) [inline scripts/archive/download-missing-blocks.sh:74]: just `help;` :)

File: scripts/archive/download-missing-blocks.sh [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+CLEAR='\033[0m'
+RED='\033[0;31m'
+
+set -u
+
+local __default_mina_network=devnet
+local __default_pg_conn=postgres://postgres@127.0.0.1:5432/archive_balances_migrated
+local __default_archive_blocks_app=mina-archive-blocks
+local __default_missing_blocks_auditor=mina-missing-blocks-auditor
+local __default_block_bucket="https://storage.googleapis.com/mina_network_block_data}"
```
Comment (emberian) [inline scripts/archive/download-missing-blocks.sh:12]:
```suggestion
local __default_block_bucket="https://storage.googleapis.com/mina_network_block_data"
```

**Review by joaosreis — APPROVED** [review]

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Review by SanabriaRusso — APPROVED** [review]

**Comment by mrmr1993** [comment]
!approved-for-mainnet

**Review by emberian — APPROVED** [review]
berkeley_migration.sh leaked into here, looks good though :)

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
Thanks a lot @emberian for noticing that

**Comment by mrmr1993** [comment]
!approved-for-mainnet

**Pull request merged**